app/cafe.py

```diff
@@ -0,0 +1,18 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not "vaccine" in visitor:
+            raise NotVaccinatedError
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError
+        elif not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError

```

Comment:
The `NotWearingMaskError` should be raised with a descriptive message. For example: `raise NotWearingMaskError('Visitor is not wearing a mask.')`.